src/test-jdk17/java/com/fasterxml/jackson/databind/tofix/RecordJsonCreatorAndJsonValue4724Test.java

```diff
@@ -1,24 +1,23 @@
 package com.fasterxml.jackson.databind.tofix;
 
+import org.junit.jupiter.api.Test;
+
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
 import com.fasterxml.jackson.databind.testutil.DatabindTestUtil;
-import com.fasterxml.jackson.databind.testutil.failure.JacksonTestFailureExpected;
-
-import org.junit.jupiter.api.Test;
 
 // [databind#4724] Deserialization behavior change with Java Records, JsonCreator and JsonValue between 2.17.2 => 2.18.0
 public class RecordJsonCreatorAndJsonValue4724Test
```

Comment:
I was thinking we should start building habit of moving fixed `/tofix` tests out of `/tofix` package :))

Reply:
We do, but I'll do that after merging PR -- there's merge conflict when moving.